usb/esp_tinyusb/include/tusb_msc_storage.h

```diff
@@ -58,6 +59,7 @@ typedef struct {
     sdmmc_card_t *card;                             /*!< Pointer to sdmmc card configuration structure */
     tusb_msc_callback_t callback_mount_changed;     /*!< Pointer to the function callback that will be delivered AFTER mount/unmount operation is successfully finished */
     tusb_msc_callback_t callback_premount_changed;  /*!< Pointer to the function callback that will be delivered BEFORE mount/unmount operation is started */
+    const esp_vfs_fat_mount_config_t *mount_config; /*!< Pointer to the FATFS mount config */
```

Comment:
Could we change this member to non-pointer?
```suggestion
    const esp_vfs_fat_mount_config_t mount_config; /*!< Pointer to the FATFS mount config */
```
It is just few bytes and it can make the user application nicer
Same on line 76